data/product.attribute.value.csv

```diff
@@ -1,75 +1,66 @@
 id,attribute_id/id,name
-product_attribute_value_02m,boxwise_WMS.product_attribute_BbA,0-2 months
-product_attribute_value_36m,boxwise_WMS.product_attribute_BbA,3-6 months
-product_attribute_value_712m,boxwise_WMS.product_attribute_BbA,7-12 months
-product_attribute_value_1318m,boxwise_WMS.product_attribute_BbA,13-18 months
-product_attribute_value_1924m,boxwise_WMS.product_attribute_BbA,19-24 months
+product_attribute_value_02m,boxwise_WMS.product_attribute_AG,0-2 months
+product_attribute_value_36m,boxwise_WMS.product_attribute_AG,3-6 months
+product_attribute_value_712m,boxwise_WMS.product_attribute_AG,7-12 months
+product_attribute_value_1318m,boxwise_WMS.product_attribute_AG,13-18 months
+product_attribute_value_1924m,boxwise_WMS.product_attribute_AG,19-24 months
+product_attribute_value_23y,boxwise_WMS.product_attribute_AG,2-3 years
+product_attribute_value_45y,boxwise_WMS.product_attribute_AG,4-5 years
+product_attribute_value_68y,boxwise_WMS.product_attribute_AG,6-8 years
+product_attribute_value_912y,boxwise_WMS.product_attribute_AG,9-12 years
+product_attribute_value_1315y,boxwise_WMS.product_attribute_AG,13-15 years
+product_attribute_value_aga,boxwise_WMS.product_attribute_AG,Adult
+product_attribute_value_agb,boxwise_WMS.product_attribute_AG,Baby
+product_attribute_value_agc,boxwise_WMS.product_attribute_AG,Child
+product_attribute_value_age,boxwise_WMS.product_attribute_AG,Everyone
 product_attribute_value_75B,boxwise_WMS.product_attribute_BS,75B
 product_attribute_value_80B,boxwise_WMS.product_attribute_BS,80B
 product_attribute_value_85B,boxwise_WMS.product_attribute_BS,85B
 product_attribute_value_90b,boxwise_WMS.product_attribute_BS,90b
 product_attribute_value_95B,boxwise_WMS.product_attribute_BS,95B
 product_attribute_value_100C,boxwise_WMS.product_attribute_BS,100C
 product_attribute_value_D,boxwise_WMS.product_attribute_BS,D+ (big)
-product_attribute_value_23y,boxwise_WMS.product_attribute_CbA,2-3 years
-product_attribute_value_45y,boxwise_WMS.product_attribute_CbA,4-5 years
-product_attribute_value_68y,boxwise_WMS.product_attribute_CbA,6-8 years
-product_attribute_value_912y,boxwise_WMS.product_attribute_CbA,9-12 years
-product_attribute_value_1315y,boxwise_WMS.product_attribute_CbA,13-15 years
 product_attribute_value_Size1,boxwise_WMS.product_attribute_DS,Size 1
 product_attribute_value_Size2,boxwise_WMS.product_attribute_DS,Size 2 (3-5 kg)
 product_attribute_value_Size3,boxwise_WMS.product_attribute_DS,Size 3 (6-9 kg)
 product_attribute_value_Size4,boxwise_WMS.product_attribute_DS,Size 4 (10-18 kg)
 product_attribute_value_Size5,boxwise_WMS.product_attribute_DS,Size 5
 product_attribute_value_Size6,boxwise_WMS.product_attribute_DS,Size 6
-product_attribute_value_Mixed,boxwise_WMS.product_attribute_MS,Mixed
-product_attribute_value_Onesize,boxwise_WMS.product_attribute_O,Onesize
+product_attribute_value_Onesize,boxwise_WMS.product_attribute_CS,Onesize
```

Comment:
Likewise, perhaps 'One size', given there are already spaces in other attribute names?

Reply:
+1